src/buildblock/Scanner.cxx

```diff
@@ -798,6 +798,53 @@ Scanner::
 set_params(Type type_v, const std::list<std::string>& list_of_names_v,
            int num_rings_v,
            int max_num_non_arccorrected_bins_v,
+           int num_detectors_per_ring_v,
+           float inner_ring_radius_v,
+           float average_depth_of_interaction_v,
+           float ring_spacing_v,
+           float bin_size_v, float intrinsic_tilt_v,
+           int num_axial_blocks_per_bucket_v, int num_transaxial_blocks_per_bucket_v, 
+           int num_axial_crystals_per_block_v, int num_transaxial_crystals_per_block_v,
+           int num_axial_crystals_per_singles_unit_v,
+           int num_transaxial_crystals_per_singles_unit_v,
+           int num_detector_layers_v,
+           float energy_resolution_v,
+           float reference_energy_v,
+           const string& scanner_geometry_v,
+           float axial_crystal_spacing_v,
+           float transaxial_crystal_spacing_v,
+           float axial_block_spacing_v,
+           float transaxial_block_spacing_v,
+           const std::string& crystal_map_file_name_v)
+{
+  set_params(type_v, list_of_names_v, num_rings_v,
+             max_num_non_arccorrected_bins_v,
+	     max_num_non_arccorrected_bins_v,
+	     num_detectors_per_ring_v, 
+	     inner_ring_radius_v, 
+             average_depth_of_interaction_v,
+             ring_spacing_v, bin_size_v, intrinsic_tilt_v,
+	     num_axial_blocks_per_bucket_v, num_transaxial_blocks_per_bucket_v,
+	     num_axial_crystals_per_block_v, num_transaxial_crystals_per_block_v,
+             num_axial_crystals_per_singles_unit_v, 
+             num_transaxial_crystals_per_singles_unit_v,
+	     num_detector_layers_v,
+             energy_resolution_v,
+             reference_energy_v,
+             scanner_geometry_v,
+             axial_crystal_spacing_v,
+             transaxial_crystal_spacing_v,
+             axial_block_spacing_v,
+             transaxial_block_spacing_v,
+             crystal_map_file_name_v);
+}
+
+
+void
+Scanner::
+set_params(Type type_v,const list<string>& list_of_names_v, 
+           int num_rings_v, 
+           int max_num_non_arccorrected_bins_v,
```

Comment:
This looked weird at the time but might be fine

Reply:
There is a build in here. There appears to be a lack of TOF variables?

Reply:
you re-inserted the `set_params()` function that was removed, so I've got rid of it again